### PR TITLE
Ignore postgres artifacts

### PR DESCRIPTION
**Pull request by atz**
These are generated when developing according to the steps in README.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,8 @@ capybara-*.html
 /tmp
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/postgres
```
Comment (jcoyne) [inline .gitignore]: We don't have a dependency on postgres. How did these get in you code?

Comment (jcoyne) [inline .gitignore]: Derp, I was thinking hyrax.

Comment (jcoyne) [inline .gitignore]: How are you getting these files here though? aren't they in you system postgres directory?

Comment (atz) [inline .gitignore]: Follow the instructions in the readme and they end up there.

**Pull request merged**